attribute-service-impl/src/main/java/org/hypertrace/core/attribute/service/model/AttributeMetadataModel.java

```diff
@@ -243,6 +246,14 @@ public void setDefinition(AttributeDefinition definition) {
     this.definition = definition;
   }
 
+  public boolean getInternal() {
+    return internal;
+  }
+
+  public void setInternal(boolean isInternal) {
+    this.internal = isInternal;
+  }
```

Comment:
since, we changed the field to `internal` now
```suggestion
  public void setInternal(boolean internal) {
    this.internal = internal;
  }
```